webgui/CMakeLists.txt

```diff
@@ -13,4 +15,4 @@ install(FILES
 # Install npm packages
 install(CODE "set(CC_PACKAGE \"${CMAKE_CURRENT_SOURCE_DIR}/package.json\")")
 install(CODE "set(INSTALL_SCRIPTS_DIR \"${INSTALL_WEBROOT_DIR}/scripts/\")")
-install(SCRIPT InstallGUI.cmake)
+install(SCRIPT InstallGUI.cmake)
```

Comment:
This is a cmake script file which `concatenates the content of user guides` into a single file. This script also `calls the doxygen command` on the concatenated file. This file is being called when you run `make install`.